documentation/src/developer_documentation/release_checklist.md

```diff
@@ -20,7 +20,7 @@ in case of absence, passing the task to a different team member.
 4. Once the changes in step 1 and step 2 have been merged, tag the commit with
    the new version number. The version tag should start with a lowercase 'v'.
    Example: v0.4.1
-5. (NOT YET APPLICABLE) Publish new crates on crates.io.
+5. Publish new crates on crates.io.
```

Comment:
This description should be extended. We can do it in a separate PR, though.

Reply:
I'm not sure if it's worth duplicating what's already written [here](https://doc.rust-lang.org/cargo/reference/publishing.html). I'll just link it.

Reply:
Let me rephrase - which crates? If there's just the `forest-filecoin`, then let's use the singular form.

Reply:
fixed in another PR #3040